core/src/epicli/data/common/defaults/configuration/feature-mapping.yml

```diff
@@ -119,9 +119,7 @@ specification:
       - firewall
       - vault
     kubernetes_master:
-      - image-registry
       - kubernetes-master
-      - repository
       - helm
```

Comment:
Isnt the Helm local repository dependent on running on the same machine where the repository is running. Isnt it using the webserver we setup their for that purpose?

Reply:
right, for now I will move Helm role to repository machine but the target might be to have Helm running without webserver maybe (and let it stay on kubernetes master)

Reply:
will use local container with tunnelling for now.